tool/generator/lib/src/projection/winrt/winrt_interface.dart

```diff
@@ -19,8 +17,14 @@ class WinRTInterfaceProjection extends ComInterfaceProjection {
       .toList()
       .join(', ');
 
-  String get relativePathToSrcDir => relativePathToSrcDirectory(File(
-      '../../lib/src/winrt/${folderFromWinRTType(typeDef.name)}/$shortName.dart'));
+  /// Returns the path to the folder where the current interface is located.
+  ///
+  /// e.g. `winrt/storage/pickers`
+  String get _currentFolderPath => 'winrt/${folderFromWinRTType(typeDef.name)}';
+
+  /// Converts [path] to an equivalent relative path from the [_currentFolderPath].
+  String _relativePathTo(String path) =>
```

Comment:
I suspect we don't need to make this function private to the library, since the whole tool is essentially private? Can we just make this `relativePathTo()`? Or is there a reason why this member in particular requires encapsulation?

Reply:
Done